src/procedures/SignFreezeIssuanceAck.ts

```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignFreezeIssuanceAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for freezing the issuance of a security token
+ */
 export class SignFreezeIssuanceAck extends Procedure<SignFreezeIssuanceAckProcedureArgs> {
   public type = ProcedureType.SignFreezeIssuanceAck;
 
+  /**
+   * - Sign an acknowledgement to freeze the issuance of the security token
```

Comment:
Sign data to confirm the intent of permanently disabling issuance of the Security Token

(delete hyphen)